src/Uno.Extensions.Reactive.Generator/Rules.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using System.Linq;
+using Microsoft.CodeAnalysis;
+
+namespace Uno.Extensions.Reactive.Generator;
+
+[SuppressMessage("ReSharper", "InconsistentNaming")]
+internal static class Rules
+{
+	// references:
+	//	Categories: https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/categories
+
+	// General usage [0000-0999]
+
+	// Bindings [1000-1999]
+
+	// Commands [2000-2999]
+	public static class FEED2001
```

Comment:
Those should be in the documentation, also, having a call to action in the message would be useful.

Reply:
I've added direct link to the doc